libsigner/src/events.rs

```diff
@@ -211,6 +213,13 @@ impl EventReceiver for SignerEventReceiver {
                 return Err(EventError::Terminated);
             }
 
+            if request.url() == "/status" {
```

Comment:
To be clear, this is a `POST` not a `GET`, right?  These are events that are `POST`ed by the node to the signer's event observer client.

Reply:
In this code it would respond to both GET and POST, though I have no problem keeping it to just GET. This is for the signer's event receiver, but the goal is to have a status endpoint for monitoring, which is why GET made sense.

This does "pollute" the event receiver, but the alternative is to have a separate endpoint/port to listen on for monitoring, if we want to have some kind of automated status check.